wsgi.py

```diff
@@ -1,19 +1,17 @@
 import os
-import multiprocessing
 from gunicorn.app.base import BaseApplication
```

Comment:
If this import is really needed then we can put it back with the following comment next to it:

`# pylint: disable=unused-import   <reason-of-the-import>`

Reply:
Looking at the gunicorn documentation and examples [Gunicorn - Custom Application](https://docs.gunicorn.org/en/stable/custom.html) it seems to me that the `import multiprocessing`  comes from an example where the `multiprocessing` module is used to compute the number of workers. In our case the number of workers is hard coded to 2. Why is this set to 2 and not like recommended to ((2 x $num_cores) + 1) in the [docu](https://docs.gunicorn.org/en/stable/design.html#how-many-workers) ?

Reply:
Oh yeah, 
Workers are Gunicorn mean of scaling horizontally, and dividing CPU use to the application
In our use case, the service is supposed to be deployed on a kubernetes cluster, which is supposed to handle horizontal scalability and in which CPU is a shared resource, The idea by setting the numbers of workers to two is for orchestrator's health check. By having two workers, we avoid false negatives in the orchestrator's health check, since having the two workers blocked at the same time (Gunicorn re-run its workers on its own), when a health check occurs is a less likely occurence. 

The logic I had was "let's leave the scaling to Kubernetes, and only get enough worker in each container to cover health checks and do their work"

 Depending on the potential number of queries expected, we could set the number of threads per worker, but that can be done in a "project to project" basis.